### PR TITLE
Fix/(not-so-small) stuff

### PR DESCRIPTION
**Pull request by carlank**

### CODE REVIEW

**Comment by carlank** [comment]
The DISCORD_TOKEN refactor will break any builds that are using the old environmental variable, but DISCORD_TOKEN is the default for discord.js, letting you not specify the token when using Client.login(). Just convenience

**Comment by carlank** [comment]
A LOT happened in this commit, due to fixing TS and eslint errors.

File: src/commands/guild/setRoles.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export = {
         const role = args[0];
         const isOwner = message.author.id === message.guild.ownerID;
         const mention = message.mentions.roles.first();
-
+        debugger
```
Comment (carlank) [inline src/commands/guild/setRoles.ts:17]:
```suggestion
```

File: src/database/index.ts [hunk 2]
```diff
@@ -15,6 +16,7 @@ mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.connection.on("error", (err) => {
     console.log(`${chalk.red.bold("[MONGOOSE]:")}${chalk.reset()} Errored.\n ${err}`);
 });
+
 mongoose.connection.on("disconnected", () => {
-    console.log(`${chalk.magenta.bold("[MONGOOSE]:]")}${chalk.reset()} Lost connection.`);
+    console.log(`${chalk.magenta.bold("[MONGOOSE]:")}${chalk.reset()} Lost connection.`);
 });
```
Comment (rbrtbrnschn) [inline src/database/index.ts:22]: change mongoURI to mongoHOST or host

File: src/database/schemas/guild.ts [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
 }
 
 const guildSchema = new Schema({
     guildId: String,
     channelId: { type: String, default: "" },
     ownerId: String,
-    roles: {},
-    tasks: []
-
+    roles: Object,
```
Comment (rbrtbrnschn) [inline src/database/schemas/guild.ts:15]: do you know why mongoose allows typing of Object but typescript say's its either deprecated currently or not best practise? just curious

Comment (carlank) [reply src/database/schemas/guild.ts:15]: So, this is a SchemaType, not a typescript type. It's a type for the purposes of the mongodb, not for typescript compilation. For ts, you can see I typed `roles` as a `Record<string, string>`

File: src/database/index.ts [hunk 4]
```diff
@@ -4,8 +4,9 @@ import { bot } from "../config";
 
 const username = process.env.MONGODB_USER || "dev";
 const password = process.env.MONGODB_PASS || "C0p5FqNEA5nV8UG7"; // provides read-only access
+const mongoURI = process.env.MONGODB_URI || 'cluster0.eit8m.mongodb.net';
```
Comment (carlank) [inline src/database/index.ts:7]:
```suggestion
const mongoHost = process.env.MONGODB_HOST || 'cluster0.eit8m.mongodb.net';
```

File: src/database/index.ts [hunk 5]
```diff
@@ -4,8 +4,9 @@ import { bot } from "../config";
...
 const db = bot.isProd ? "main" : "dev";
-const uri = `mongodb+srv://${username}:${password}@cluster0.eit8m.mongodb.net/${db}?retryWrites=true&w=majority`;
+const uri = `mongodb+srv://${username}:${password}@${mongoURI}/${db}?retryWrites=true&w=majority`;
```
Comment (carlank) [inline src/database/index.ts:9]:
```suggestion
const uri = `mongodb+srv://${username}:${password}@${mongoHost}/${db}?retryWrites=true&w=majority`;
```

**Review by rbrtbrnschn — APPROVED** [review]

**Review by carlank — COMMENTED** [review]
Change mongoURI to mongoHost

Oh, this failed because it was post-merge. I'll make the change elsewhere

**Pull request merged**